services/dkg/pedersen/mod.go

```diff
@@ -364,6 +370,9 @@ func (a *Actor) ComputePubshares() error {
 	}
 
 	players := mino.NewAddresses(a.handler.startRes.GetParticipants()...)
+	if players.Len() == 0 {
+		return xerrors.Errorf("the list of Participants is empty")
+	}
```

Comment:
should we call a.setErr here ? 
This function is called when we want to decrypt the ballots, so I am wondering what we would be able to do / should do in the scenario where players is empty...

Reply:
yes, true.
@Flamewind97 please add a `setErr` as in other cases.

Reply:
In the end we don't need, I mixed the context.